scripts/sidecarScriptApi.ts

```diff
@@ -103,12 +103,17 @@ export const launchProcess = (
 		procs[proc].stderr.on('data', (data: Buffer) => {
 			console.error(data.toString().trim());
 
-			if (resolverStartupErr && data.toString().includes(resolverStartupErr)) {
+			if (resolverJestErr && data.toString().trim().includes(resolverJestErr)) {
+				jestStatus = Failed;
+			}
+
+			if (resolverStartupErr && data.toString().trim().includes(resolverStartupErr)) {
 				resolve(Failed);
 			}
```

Comment:
Just want to understand this bit; if there's a startup error and `resolverStartupErr` is true then we bail immedaitely with Failed, but in the `resolverJestErr` case we just set it to failed and let the "close" handle it?

Reply:
(I guess I'm wondering why we choose to immediately resolve in some cases, and wait for close in others?)

Reply:
Yea the difference here being when we know Jest fails, we want it to finish resolving the suite, but acknowledge it's failure. Whereas if there is a startup error, we just resolve the process. This makes sure the Jest finishes doing whatever it needs before we log its failed tests.

Reply:
Ok gotcha, makes sense!